src/style.css

```diff
@@ -1,3 +1,71 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+}
+
+.html {
+  --header-link-color: #00acdc;
+}
+
+.body {
+  margin: 0;
+}
+
+.img {
+  display: block;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  font-family: Roboto, sans-serif;
```

Comment:
You should use fallback fonts. Add an alternative font-family in case the main one doesn't work. (CHECKLIST: Remember to use fallback fonts - alternative font-family in case the main one doesn't work)